README.md

```diff
@@ -2,35 +2,33 @@
 
-[![CircleCI](https://img.shields.io/circleci/build/github/auth0/express-openid-connect/master?style=flat-square)](https://circleci.com/gh/auth0/express-openid-connect/tree/master)
-[![codecov](https://img.shields.io/codecov/c/github/auth0/express-openid-connect?style=flat-square)](https://codecov.io/gh/auth0/express-openid-connect)
-[![NPM version](https://img.shields.io/npm/v/express-openid-connect.svg?style=flat-square)](https://npmjs.org/package/express-openid-connect)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fexpress-openid-connect.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fexpress-openid-connect?ref=badge_shield)
-
-## Table of Contents
-
-- [Documentation](#documentation)
-- [Install](#install)
-- [Getting Started](#getting-started)
-- [Architecture](./ARCHITECTURE.md)
-- [Contributing](#contributing)
-- [Troubleshooting](./TROUBLESHOOTING.md)
-- [FAQs](./FAQ.md)
-- [Support + Feedback](#support--feedback)
-- [Vulnerability Reporting](#vulnerability-reporting)
-- [What is Auth0](#what-is-auth0)
-- [License](#license)
+![Release](https://img.shields.io/npm/v/express-openid-connect)
+[![Codecov](https://img.shields.io/codecov/c/github/auth0/express-openid-connect)](https://codecov.io/gh/auth0/express-openid-connect)
+![Downloads](https://img.shields.io/npm/dw/express-openid-connect)
+[![License](https://img.shields.io/:license-mit-blue.svg?style=flat)](https://opensource.org/licenses/MIT)
+![CircleCI](https://img.shields.io/circleci/build/github/auth0/express-openid-connect)
+
+:books: [Documentation](#documentation) - :rocket: [Getting Started](#getting-started) - :computer: [API Reference](#api-reference) - :speech_balloon: [Feedback](#feedback)
 
 ## Documentation
 
-- Our [Express Quickstart](https://auth0.com/docs/quickstart/webapp/express) is the quickest way to get up and running from scratch.
-- Use the [Examples for common configurations](https://github.com/auth0/express-openid-connect/blob/master/EXAMPLES.md) for use cases beyond the basics.
-- The [API documentation](https://auth0.github.io/express-openid-connect) details all configuration options, methods, and data that this library provides.
-- You can [run the sample application](https://github.com/auth0-samples/auth0-express-webapp-sample/tree/master) to see how this SDK functions without writing your own integration.
+- [Quickstart](https://auth0.com/docs/quickstart/webapp/express) - our guide for quickly adding Auth0 to your Express app.
+- [Sample](https://github.com/auth0-samples/auth0-express-webapp-sample/tree/master/01-Login) - an Express app integrated with Auth0.
+- [FAQs](./FAQ.md) - Frequently asked questions about express-openid-connect.
+- [Docs Site](https://auth0.com/docs) - explore our docs site and learn more about Auth0.
+- [Docs Site(https)
+
+## Getting Started
+
+### Requirements
+
+This library supports the following tooling versions:
+
+* Node.js `^10.19.0 || >=12.0.0`
```

Comment:
Both 10 and 12 are not in LTS anymore, even though we might be compatible, would it make sense to update this to mention the current LTS versions, assuming we support these?

Doesn't have to be in this PR, I just noticed it and wanted to raise it.

Reply:
Yeah it didn't feel right having 10 and 12 here, but given that we only run tests against 12 currently I didn't want to add 14/16/18 without having at least one of them in CI

Reply:
Fair point, let's leave this for now.

Reply:
Yeah, that's out of date - will fix that